setup.cfg

```diff
@@ -257,6 +257,7 @@ jenkins_jobs.wrappers =
     release=jenkins_jobs.modules.wrappers:release
     rvm-env=jenkins_jobs.modules.wrappers:rvm_env
     sauce-ondemand=jenkins_jobs.modules.wrappers:sauce_ondemand
+    sonar-prepare=jenkins_jobs.modules.wrappers:sonar_prepare
```

Comment:
i dont know :) ill make it whatever
just figured sonar might make it a bit confusing, like there was some configuration available, but it just a check box